test/ussd_popi_user_data.test.js

```diff
@@ -0,0 +1,354 @@
+                                {session_event: "new"}
+                                , "3" // pick option 3
+                                , "2" // pick no
+                            )
+                            .check.interaction({
+                                state: "state_language_not_changed",
+                                reply: [
+                                    "Your language preference stored on " +
+                                    "MomConnect has not been changed."
+                                ].join('\n')
+                            })
+                            .check(function(api) {
+                                utils.check_fixtures_used(api, [1, 4, 121]);
+                            })
+                            .run();
+                        });
+                    });
+                });
+
+                describe("user chooses to delete data", function() {
+                    it("should go to state_delete_data", function() {
+                        return tester
+                        .setup.user.addr("27820001002")
+                        .inputs(
+                            {session_event: "new"}
+                            , "4" // pick option 4
+                        )
+                        .check.interaction({
+                            state: "state_delete_data",
+                            reply: [
+                                "Are you sure you would like to permanently " +
+                                "delete all personal information stored on MomConnect? ",
+                                "1. Yes",
+                                "2. No"
+                            ].join('\n')
+                        })
+                        .check(function(api) {
+                            utils.check_fixtures_used(api, [1, 4, 121]);
+                        })
+                        .run();
+                    });
+                    describe("user chooses yes", function() {
+                        it("should go to state_info_deleted", function() {
+                            return tester
+                            .setup.user.addr("27820001002")
+                            .inputs(
+                                {session_event: "new"}
+                                , "4" // pick option 4
+                                , "1" // pick yes
+                            )
+                           
+                            .check.interaction({
+                                state: "state_info_deleted",
+                                reply: [
+                                    'Thank you. Your personal information stored on ' +
+                                    'MomConnect has been permanently removed.'
+                                ].join('\n')
+                            })
+                            .check(function(api) {
+                                // later check that actually has been removed
```

Comment:
A common format for these sorts of comments is `// TODO: check that actually has been removed`
Some IDEs will find those comments and make a To Do list from them